server.py

```diff
@@ -1,105 +0,0 @@
-import sys
```

Comment:
same here

Reply:
i actually  deleted it, it's in the red color